daprdocs/content/en/operations/performance-and-scalability/_index.md

```diff
@@ -0,0 +1,80 @@
+---
```

Comment:
Given this is a new section under operations, the guideline even for one article is to have a _index.md file that has a general heading like this and this article is specifically performance for the service invocation API. Hence split this into 2 documents.

Reply:
Done